platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -56,6 +62,8 @@
 
   private static final String INVALID_NULL_EXPORT_DIR = "invalid null export directory";
 
+  @VisibleForTesting static final Path RESTART_JVM = Paths.get("bin", "restart.jvm");
```

Comment:
❗️ We should specify the root of this path at `$ddf.home` instead of using a relative path.

Reply:
That is actually a left over from a previous implementation attempt. It was being resolved from ddf_home at the time when used. I have removed the line.